content/v2.0/reference/api/client-libraries/js.md

```diff
@@ -1,12 +1,167 @@
 ---
-title: InfluxDB JavaScript client library
+title: JavaScript client library
 list_title: JavaScript
-description: Use the JavaScript client library to interact with InfluxDB.
-external_url: https://github.com/influxdata/influxdb-client-js
+description: >
+  Use the JavaScript client library to interact with InfluxDB.
 menu:
   v2_0_ref:
     name: JavaScript
     parent: Client libraries
-    url: https://github.com/influxdata/influxdb-client-js
+v2.0/tags: [client libraries, JavaScript]
+aliases:
+  - /v2.0/reference/api/client-libraries/js-cl-guide/
 weight: 201
 ---
+
+Use the [InfluxDB JavaScript client library](https://github.com/influxdata/influxdb-client-js) to integrate InfluxDB into JavaScript scripts and applications.This client supports both browser environments and NodeJS environments. 
+
+This guide presumes some familiarity with JavaScript, browser environments, and InfluxDB.
+If just getting started, see [Getting started with InfluxDB](/v2.0/get-started/).
+
+## Before you begin
+
+1. Install NodeJS:
+
+    ```sh
+    brew install node 
+    ``` 
+2. Ensure that InfluxDB is running.
+   If running InfluxDB locally, visit http://localhost:9999.
+   (If using InfluxDB Cloud, visit the URL of your InfluxDB Cloud UI.
+   For example: https://us-west-2-1.aws.cloud2.influxdata.com.)
+
+## Easiest way to get started 
+1. clone the [examples directory](https://github.com/influxdata/influxdb-client-js/tree/master/examples) in the [influxdb-client-js](https://github.com/influxdata/influxdb-client-js) repo. 
+2. Navigate to examples directory and install NPM
+    ```sh
+    cd examples 
+    npm install
+    ```
```

Comment:
We can show them how to use `npm` or `yarn` using some of the shortcodes in the docs:

_I know the indentation looks messy, but this is how it has to be done to get the code tabs to format correctly when nested in an ordered list._

~~~
2. Navigate to examples directory and install NPM

    {{< code-tabs-wrapper >}}
    {{% code-tabs %}}
[npm](#)
[yarn](#)
    {{% /code-tabs %}}
    {{% code-tab-content %}}
    ```sh
# Navigate into examples directory
cd examples 

# Install dependencies
npm install
    ```
    {{% /code-tab-content %}}
    {{% code-tab-content %}}
    ```sh
# Navigate into examples directory
cd examples 

# Install dependencies
yarn install
    ```
    {{% /code-tab-content %}}
    {{< /code-tabs-wrapper >}}
~~~